README.md

```diff
@@ -47,18 +47,84 @@ luarocks install --server=https://luarocks.org/dev luaformatter
+
+      --break-after-functioncall-lp          Break after '(' of function call
+      --no-break-after-functioncall-lp       Do not break after '(' of functioncall
+
+      --break-before-functioncall-rp         Break before ')' of function call
+      --no-break-before-functioncall-rp      Do not break before ')' of functioncall
+
+      --align-parameter                      Align the parameters
+      --no-align-parameter                   Do not align the parameters
+
+      --chop-down-parameter                  Chop down all parameters
+      --no-chop-down-parameter               Do not chop down all parameters
+
+      --break-after-functiondef-lp           Break after '(' of function def
+      --no-break-after-functiondef-lp        Do not break after '(' of functiondef
+
+      --break-before-functiondef-rp          Break before ')' of function def
+      --no-break-before-functiondef-rp       Do not break before ')' of functiondef
+
+      --align-table-field                    Align fields of table
+      --no-align-table-field                 Do not align fields of table
+
+      --break-after-table-lb                 Break after '{' of table
+      --no-break-after-table-lb              Do not break after '{' of table
+
+      --break-before-table-rb                Break before '}' of table
+      --no-break-before-table-rb             Do not break before '}' of table
+
+      --chop-down-table                      Chop down any table
+      --no-chop-down-table                   Do not chop down any table
+
+      --chop-down-kv-table                   Chop down table if table contains key
+      --no-chop-down-kv-table                Do not chop down table if table contains key
+
+      --extra-sep-at-table-end               Add a extra field separator
+      --no-extra-sep-at-table-end            Do not add a extra field separator
+
+      --break-after-operator                 Put break after operators
+      --no-break-after-operator              Do not put break after operators
+
+      --double-quote-to-single-quote         Transform string literals to use single quote
+      --no-double-quote-to-single-quote      Do not transform string literals to use single quote
+
+      --single-quote-to-double-quote         Transform string literals to use double quote
+      --no-single-quote-to-double-quote      Do not transform string literals to use double quote
+
       "--" can be used to terminate flag options and force all following
       arguments to be treated as positional options
 ```
 
 The program will attempt to automatically use the current directory's `.lua-format` file if no config file is passed in the command line. On Linux it will use `$XDG_CONFIG_HOME/luaformatter/config.yaml` if `.lua-format` does not exist.
-In case there's no file, it will fallback to the default configuration.
+In case it doesn't find any suitable file it will check commandline configuration value and then fallback to the default values.\
```

Comment:
Restore to the original because the behavior you described is not the actual one implemented.
The below phrase is the appropriate one.

Reply:
ok